src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC } from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem';
+
+interface Props {
+  todos: Todo[];
+}
+
+export const TodoList: FC<Props> = ({ todos }) => {
+  return (
+    <section className="todoapp__main">
+
+      {
```

Comment:
Wrap the todos.map inside a conditional rendering to hide the list and footer if there are no todos yet.